decode.rb

```diff
@@ -0,0 +1,57 @@
+MORSE_CODE = {
+  '.-' => 'A',
+  '-...' => 'B',
+  '-.-.' => 'C',
+  '-..' => 'D',
+  '.' => 'E',
+  '..-.' => 'F',
+  '--.' => 'G',
+  '....' => 'H',
+  '..' => 'I',
+  '.---' => 'J',
+  '-.-' => 'K',
+  '.-..' => 'L',
+  '--' => 'M',
+  '-.' => 'N',
+  '---' => 'O',
+  ' .--.' => 'P',
+  '--.-' => 'Q',
+  '.-.' => 'R',
+  '...' => 'S',
+  '-' => 'T',
+  '..-' => 'U',
+  '...-' => 'V',
+  '.--' => 'W',
+  '-..-' => 'X',
+  '-.--' => 'Y',
+  ' --..' => 'Z'
+}.freeze
+
+def decode_char(str)
+  if MORSE_CODE.key?(str)
+    MORSE_CODE[str]
+  else
+    ''
+  end
+end
+
+def decode_word(str)
+  word = ''
+  str.split.each do |letter|
+    word += MORSE_CODE[letter]
+  end
+  word
+end
+
+def decode(str)
+  message = ''
+  str.split('   ').each do |word|
+    message += "#{decode_word(word)} "
+  end
+  message
```

Comment:
- [ ] From the image below, you can see there trailing spaces at the end of your sentences. You can fix that returning `message.strip`, instead of `message` alone.

  ![fgh](https://user-images.githubusercontent.com/54848363/178838376-a2cba5ea-38c9-49ee-af40-4909955a1c5b.jpg)

Reply:
Thank you for your recommendation